src/main/webapp/app/course/competencies/import-standardized-competencies/course-import-standardized-prerequisites.component.html

```diff
@@ -0,0 +1,114 @@
+            @if (selectedCompetency) {
+                <div style="background-color: var(--overview-light-background-color)" class="card d-flex flex-grow-1 w-100 h-100 p-3 ms-1">
+                    <jhi-standardized-competency-detail
+                        [competency]="selectedCompetency"
+                        [sourceString]="sourceString"
+                        [knowledgeAreaTitle]="selectedCompetency.knowledgeAreaTitle ?? ''"
+                        (onClose)="closeCompetencyDetails()"
+                    />
+                </div>
+            }
+        </div>
+    } @else if (isLoading) {
+        <div class="d-flex justify-content-center">
+            <div class="spinner-border" role="status">
+                <span class="sr-only" jhiTranslate="loading"></span>
+            </div>
+        </div>
+    } @else {
+        <span jhiTranslate="artemisApp.standardizedCompetency.courseImport.empty"></span>
+    }
+    <h3 class="mt-3" jhiTranslate="artemisApp.standardizedCompetency.courseImport.selectedTitle"></h3>
+    @if (selectedCompetencies.length) {
+        <table class="table table-striped align-middle flex">
+            <thead (sortChange)="sortSelected($event)" class="thead-dark" jhiSort>
+                <tr class="flex-row">
+                    <th jhiSortBy="id" class="col-0.5">
+                        <span>#</span>
+                        <fa-icon [icon]="faSort" />
+                    </th>
+                    <th jhiSortBy="title" class="col-2">
+                        <span [jhiTranslate]="'artemisApp.standardizedCompetency.model.title'"></span>
+                        <fa-icon [icon]="faSort" />
+                    </th>
+                    <th class="col">
+                        <span [jhiTranslate]="'artemisApp.standardizedCompetency.model.description'"></span>
+                    </th>
+                    <th jhiSortBy="knowledgeAreaTitle" class="col-2">
+                        <span [jhiTranslate]="'artemisApp.standardizedCompetency.model.knowledgeArea'"></span>
+                        <fa-icon [icon]="faSort" />
+                    </th>
+                    <!-- column for buttons -->
+                    <th class="col-1"></th>
+                </tr>
+            </thead>
+            <tbody>
+                @for (competency of selectedCompetencies; track competency.id) {
+                    <tr>
+                        <td class="col-0.5">{{ competency.id }}</td>
+                        <td class="col-2">{{ competency.title }}</td>
+                        <td class="markdown-preview col" [innerHTML]="competency.description | htmlForMarkdown"></td>
+                        <td class="col-2">{{ competency.knowledgeAreaTitle }}</td>
+                        <td class="col-1">
+                            <jhi-button [btnType]="ButtonType.ERROR" [icon]="faTrash" (onClick)="deselectCompetency(competency)" />
+                        </td>
+                    </tr>
```

Comment:
same here you could use the @empty block to get rid of additional if/else to check if the list is empty